openeo/Pre&PostCovid.ipynb

```diff
@@ -0,0 +1,358 @@
+      "text/plain": [
+       "<Figure size 640x480 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax1 = plt.subplots(dpi=100)\n",
+    "ax1.plot(predata.t, predata.NO2.to_numpy().flatten(), color=\"g\")\n",
+    "ax1.set_xlabel(\"During COVID\")\n",
+    "ax2 = ax1.twiny()\n",
+    "ax2.plot(postdata.t, postdata.NO2.to_numpy().flatten(), color=\"r\")\n",
+    "ax2.set_xlabel(\"Post COVID\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5f459e68-b702-4dd8-a07b-79d5f285b6b9",
+   "metadata": {},
+   "source": [
+    "The red line in the post illustrates the NO2 concentration duration during COVID, and the green line represents the period after COVID for similar months of various years. Although the shift is not substantial, there's a minor decline in NO2 levels. \n",
```

Comment:
```suggestion
    "The red line in the plot shows the NO2 concentration duration post COVID, and the green line represents the period during COVID for similar months of various years. Although the shift is not substantial, there's a minor decline in NO2 levels. \n",
```

I would also switch green and red subsentences

Reply:
I realised that earlier, I had made mistakes in presenting the result. Logically, it should decrease when no human activities going on.

Also switch the green and red representation as suggested(red-during, gree-post)